src/Tests/No_UTF8_BOM.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.IO;
+using System.Text;
+using NServiceBus.MessageInterfaces.MessageMapper.Reflection;
+using NServiceBus.Newtonsoft.Json;
+using NUnit.Framework;
+
+[TestFixture]
+public class No_UTF8_BOM
```

Comment:
Let's rename this to `When_serializing_a_message`

Reply:
Done.